### DISCUSSION TITLE
jsr305 compile-time dependency

### DISCUSSION THREAD

**Discussion by gnahraf** [LibrePDF/OpenPDF#820] [Q&A] [answered]
I'm running into an obstacle or 2 trying to build a jlink'ed or native-image for my apps that use this library. As a workaround, I removed the com.google.code.findbugs:jsr305 jar and things still work fine: I never hit a ClassNotFoundException at runtime.

Curiously, the `module-info.java` does not *require* jsr305; it's only in the POM file. (So I'm wondering how can that be, and maybe just an oversight in the pom.)

So my question is a) Why is it there? and b) Can I expect to run into trouble with my workaround?

**Comment by asturio** [comment, marked as answer]
Deprecated. jsr305 was removed in 1.3.31
